<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>svntogit/community.git - Git clone of the 'community' subversion repository</title>
<meta name='generator' content='cgit v0.8.3.4-124-ge7dc'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit.css'/>
<link rel='alternate' title='Atom feed' href='http://projects.archlinux.org/svntogit/community.git/atom/spidermonkey/trunk/spidermonkey-Makefile.patch?h=master' type='application/atom+xml'/>
<link rel="stylesheet" href="/archnavbar.css" />
<link rel="icon" href="/favicon.ico" type="image/x-icon" />
<link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
</head>
<body>
	<div id="archnavbar"><!-- Arch Linux global navigation bar -->
		<div id="archnavbarlogo">
			<p><a href="http://www.archlinux.org/" title="Arch news, packages, projects and more"></a></p>
		</div>
		<div id="archnavbarmenu">
			<ul id="archnavbarlist">
				<li id="anb-home"><a href="http://www.archlinux.org/" title="Arch news, packages, projects and more">Home</a></li>
				<li id="anb-packages"><a href="http://www.archlinux.org/packages/" title="Arch Package Database">Packages</a></li>
				<li id="anb-forums"><a href="http://bbs.archlinux.org/" title="Community forums">Forums</a></li>
				<li id="anb-wiki"><a href="http://wiki.archlinux.org/" title="Community documentation">Wiki</a></li>
				<li id="anb-bugs"><a href="http://bugs.archlinux.org/" title="Report and follow bugs">Bugs</a></li>
				<li id="anb-aur"><a href="http://aur.archlinux.org/" title="Arch Linux User Repository">AUR</a></li>
				<li id="anb-download"><a href="http://www.archlinux.org/download/" title="Get Arch Linux">Download</a></li>
			</ul>
		</div>
	</div><!-- #archnavbar -->
<div id='cgit'><table id='header'>
<tr>
<td class='main'><a href='/'>index</a> : <a title='svntogit/community.git' href='/svntogit/community.git/'>svntogit/community.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>Git clone of the 'community' subversion repository</td><td class='sub right'>Arch Linux Trusted Users</td></tr></table>
<table class='tabs'><tr><td>
<a href='/svntogit/community.git/'>summary</a><a href='/svntogit/community.git/refs/'>refs</a><a href='/svntogit/community.git/log/spidermonkey/trunk/spidermonkey-Makefile.patch'>log</a><a class='active' href='/svntogit/community.git/tree/spidermonkey/trunk/spidermonkey-Makefile.patch'>tree</a><a href='/svntogit/community.git/commit/spidermonkey/trunk/spidermonkey-Makefile.patch'>commit</a><a href='/svntogit/community.git/diff/spidermonkey/trunk/spidermonkey-Makefile.patch'>diff</a><a href='/svntogit/community.git/stats/spidermonkey/trunk/spidermonkey-Makefile.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/svntogit/community.git/log/spidermonkey/trunk/spidermonkey-Makefile.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/svntogit/community.git/tree/'>root</a>/<a href='/svntogit/community.git/tree/spidermonkey'>spidermonkey</a>/<a href='/svntogit/community.git/tree/spidermonkey/trunk'>trunk</a>/<a href='/svntogit/community.git/tree/spidermonkey/trunk/spidermonkey-Makefile.patch'>spidermonkey-Makefile.patch</a></div><div class='content'>blob: abf1f31640e5e22c104cc5852c598ab7fac23a77 (<a href='/svntogit/community.git/plain/spidermonkey/trunk/spidermonkey-Makefile.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a class='no' id='n1' name='n1' href='#n1'>1</a>
<a class='no' id='n2' name='n2' href='#n2'>2</a>
<a class='no' id='n3' name='n3' href='#n3'>3</a>
<a class='no' id='n4' name='n4' href='#n4'>4</a>
<a class='no' id='n5' name='n5' href='#n5'>5</a>
<a class='no' id='n6' name='n6' href='#n6'>6</a>
<a class='no' id='n7' name='n7' href='#n7'>7</a>
<a class='no' id='n8' name='n8' href='#n8'>8</a>
<a class='no' id='n9' name='n9' href='#n9'>9</a>
<a class='no' id='n10' name='n10' href='#n10'>10</a>
<a class='no' id='n11' name='n11' href='#n11'>11</a>
<a class='no' id='n12' name='n12' href='#n12'>12</a>
<a class='no' id='n13' name='n13' href='#n13'>13</a>
<a class='no' id='n14' name='n14' href='#n14'>14</a>
<a class='no' id='n15' name='n15' href='#n15'>15</a>
<a class='no' id='n16' name='n16' href='#n16'>16</a>
<a class='no' id='n17' name='n17' href='#n17'>17</a>
<a class='no' id='n18' name='n18' href='#n18'>18</a>
<a class='no' id='n19' name='n19' href='#n19'>19</a>
<a class='no' id='n20' name='n20' href='#n20'>20</a>
<a class='no' id='n21' name='n21' href='#n21'>21</a>
<a class='no' id='n22' name='n22' href='#n22'>22</a>
</pre></td>
<td class='lines'><pre><code>*** Makefile.ref.orig	2009-02-16 09:56:29.000000000 -0600
<span class="hl kwb">--- Makefile.ref	2009-02-16 09:58:59.000000000 -0600</span>
***************
*** 169,184 ****
<span class="hl kwb">--- 169,185 ----</span>
  	jstypes.h	\
  	jsprvtd.h	\
  	jspubtd.h	\
  	jsregexp.h	\
  	jsscan.h	\
  	jsscope.h	\
  	jsscript.h	\
  	jsstr.h		\
<span class="hl kwa">+ 	jsutil.h        \</span>
  	jsxdrapi.h	\
  	jsxml.h		\
  	$(NULL)
  
  API_HFILES =		\
  	jsapi.h		\
  	jsdbgapi.h	\
  	$(NULL)
</code></pre></td></tr></table>
</div> <!-- class=content -->
</div>
<div class="foot" style="padding-left:1em;padding-right:1em;">
<p>Copyright &copy; 2002-2010 <a href="mailto:jvinet@zeroflux.org"
title="contact Judd Vinet">Judd Vinet</a> and <a href="mailto:aaron@archlinux.org"
title="contact Aaron Griffin">Aaron Griffin</a>. The Arch Linux name and logo
are recognized trademarks. Some rights reserved. The registered trademark
Linux&copy; is used pursuant to a sublicense from LMI, the exclusive licensee
of Linus Torvalds, owner of the mark on a world-wide basis.</p>
</div>
</div> <!-- id=cgit -->
</body>
</html>
